nevergrad/functions/PBOfunctions/core.py

```diff
@@ -0,0 +1,108 @@
+# from IOHexperimenter import IOH_function 
+#TODO: figure out which way is better. importing the pbo_dict ignores all wrapper-functions, but it seems
+#like they are not needed here anyway, only the evaluate is actually used, so this should be faster
+#But, is is quite an ugly solution and should probably be improved
+from IOHexperimenter.src.IOH_function import pbo_fid_dict 
+from IOHexperimenter import W_Model_OneMax, W_Model_LeadingOnes
+import numpy as np
+import nevergrad as ng
+from nevergrad.common.typetools import ArrayLike
+from .. import base
+
+class PBO_function(base.ExperimentFunction):
```

Comment:
Please use standard pep8 names, than would mean `PBOFunction`, or `PboFunction`
similarly, the folder name should be `pbofunctions`, or better yet probably `pbo` since the subpackage only contains functions